presto-product-tests/conf/docker/singlenode-kerberized/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
     command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode-kerberized run
     volumes:
       - ../../../../:/docker/volumes/presto
-      - ../../../../presto-product-tests/conf/presto/etc/hive-kerberized.properties:/docker/volumes/presto/presto-product-tests/conf/presto/etc/catalog/hive.properties
+      - ${PRESTO_HIVE_CONFIG}:/docker/volumes/presto/presto-product-tests/conf/presto/etc/catalog/hive.properties
```

Comment:
Maybe better create a new docker-compose instead of using `if` + environment variable